LPD-40364 Upload files: Drag & Drop

oliv-yu opened the pull request:
### What is this trying to solve?
https://liferay.atlassian.net/browse/LPD-40364 - Upload files: Drag & Drop

### How am I fixing it?
Basically applies the changes from `frontendDataSet`'s new drop interactions https://liferay.atlassian.net/browse/LPD-44645 in order to allow drag and drop for files in the All, Files, and Folder sections. A set of `fileDropSettings` were added into the corresponding propsTransformer. Also the corresponding language key was updated to mention dropping in files!

This requires a review from frontend infrastructure (pending [here](https://github.com/liferay-frontend/liferay-portal/pull/5045)) so keeping this as a draft for now! It might be that FI would need to update how they want to do callbacks.

### How can you verify that it works?
Feel free to deploy and try it out, and here's a short screenshare of using it:

https://github.com/user-attachments/assets/ef324c9e-f112-469e-ad16-32b2f105e8b5

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-28

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-28.liferay.com/job/test-portal-source-format/8403">test-portal-source-format#8403</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-28.liferay.com/userContent/jobs/test-portal-source-format/builds/8403/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6999">liferay-content-management#6999</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/296659144">[master] ci:test:sf - oliv-yu &gt; liferay-content-management - PR#6999 - 2025-08-20[16:51:07]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>296659144</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-28.liferay.com/job/test-portal-source-format/8403/">test-portal-source-format#8403</a><br/></div></div>

oliv-yu commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-29.liferay.com/job/test-portal-source-format/7124/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/7124/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><p><span>Ran com.liferay.source.formatter at released version <a href="https://repository.liferay.com/nexus/content/repositories/liferay-public-releases/com/liferay/com.liferay.source.formatter/1.0.1528">1.0.1528</a>.</span><br/><em>*The <a href="https://github.com/liferay/liferay-portal/commits/master/modules/util/source-formatter">latest version</a> has not been released.</em></p><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/891bcb4b122891b4e04a56d7e47c3263d5364f6e">891bcb4b122891b4e04a56d7e47c3263d5364f6e</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/oliv-yu/liferay-portal/tree/LPD-40364-cms-fds-dropzone-combined">LPD-40364-cms-fds-dropzone-combined</a><br/>Branch GIT ID: <a href="https://github.com/oliv-yu/liferay-portal/commit/e456ac4083866dc4386083f4c7020d1cee2bb0b8">e456ac4083866dc4386083f4c7020d1cee2bb0b8</a></p>1 out of 1 jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-source-format/7124/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/7124/jenkins-report.html">here</a>.</h5></details></html>

ambrinchaudhary reviewed (approved):
Great job here @oliv-yu, I am very happy that the"contract" between FI and CM works and to see this working 🫶 🎉 

Let's wait for FI to review their part, but frontend changes in our code LGTM.

ambrinchaudhary commented:
Any way to add tests to cover this?

ambrinchaudhary commented:
> hey @oliv-yu ,
> 
> Can I review this already ? I see it in a draft status, that's why I am asking.

The draft is because the first commit is in FI queue for review and we cannot forward this before, but I think the rest is finished and working, so you can review it, thanks Mikel!

Mikellorza commented:
@oliv-yu 

also, I am seeing the key and not the translation here:

<img width="3794" height="1590" alt="image" src="https://github.com/user-attachments/assets/03686199-51b9-44f1-9f46-bd5da97faae9" />

oliv-yu commented on modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/ViewFilesSectionDisplayContext.java:
```diff
@@ -67,7 +67,8 @@ public Map<String, Object> getEmptyState() {
 			LanguageUtil.get(
-				httpServletRequest, "click-new-to-create-your-first-file")
+				httpServletRequest,
+				"click-new-or-drag-and-drop-your-files-here")
```
Sounds good! I removed the original language key, rebased to avoid conflicts, and ran `buildLang` 👍 let me know if there's any issues seeing the new text!

liferay-continuous-integration commented:
Test suite cms has been triggered on http://test-1-37

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-28

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>cms</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/23994/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/23994/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/23994">test-portal-acceptance-pullrequest(master)#23994</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/23994/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>cms<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6999">liferay-content-management#6999</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/300168840">[master] ci:test:cms - oliv-yu &gt; liferay-content-management - PR#6999 - 2025-08-22[13:03:15]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>300168840</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/23994/">test-portal-acceptance-pullrequest(master)#23994</a><br/></div></div>

oliv-yu commented:
@Mikellorza Oh no, I think there's a few tests failing because of the new `additionalProps`. I'm not too familiar with integration tests, would you be able to help me with that? 😅

Mikellorza commented:
Hi @oliv-yu ,

Please, add my commit to your branch from https://github.com/oliv-yu/liferay-portal/pull/107/commits

Regards!

oliv-yu commented:
ci:test:cms

liferay-continuous-integration commented:
Test suite cms has been triggered on http://test-1-39

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/14421">test-portal-acceptance-pullrequest(master)#14421</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14421/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>cms<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6999">liferay-content-management#6999</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/301236049">[master] ci:test:cms - oliv-yu &gt; liferay-content-management - PR#6999 - 2025-08-25[08:45:26]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>301236049</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/14421/">test-portal-acceptance-pullrequest(master)#14421</a><br/></div></div>

Mikellorza commented:
ci:test:cms

liferay-continuous-integration commented:
Test suite cms has been triggered on http://test-1-27

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>cms</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/14884/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14884/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:cms - 3 out of 5 jobs passed in 49 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ae5aa6fe52ef87890e115bd346805c9d85b47d6c">ae5aa6fe52ef87890e115bd346805c9d85b47d6c</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ae5aa6fe52ef87890e115bd346805c9d85b47d6c">ae5aa6fe52ef87890e115bd346805c9d85b47d6c</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/299791050">EE Development Acceptance (master) - 1608 - 2025-08-22[08:34:37]</a></p><details><summary><strong>ci:test:cms - 3 out of 5 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/14884/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>js-unit<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1212558/">js-unit/0/0</a></li></ul></li><li>modules-integration-postgresql163<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1239894/"><strike>modules-integration-postgresql163/0/0</strike></a></li></ul></li><li>playwright-js-tomcat101-postgresql163<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1490677/">playwright-js-tomcat101-postgresql163/0/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1307038/">playwright-js-tomcat101-postgresql163/1/0</a></li></ul></li></ul></div><details><summary><strong>3 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14884/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1239894//consoleText">modules-integration-postgresql163/0/0</a><ol><li><details><summary><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1239894/testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewRecycleBinSectionDisplayContextTest">com.liferay.site.cms.site.initializer.internal.display.context.test.ViewRecycleBinSectionDisplayContextTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/300024044/history">Failed 0 of last 24</a></summary><div><ol><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1239894//testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewRecycleBinSectionDisplayContextTest/testGetAdditionalProps">testGetAdditionalProps</a></li></ol></div></details></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/14884">test-portal-acceptance-pullrequest(master)#14884</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14884/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>cms<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6999">liferay-content-management#6999</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/301277359">[master] ci:test:cms - oliv-yu &gt; liferay-content-management - PR#6999 - 2025-08-25[09:43:17]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>301277359</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/14884/">test-portal-acceptance-pullrequest(master)#14884</a><br/></div></div>

Mikellorza commented:
Hey @oliv-yu ,

I see there is another test failing. I will fix it tomorrow,  and I will send the fix to you.

Mikellorza commented:
@oliv-yu there are some conflicts. Could you please fix them in order to fix the other test that fails?

[com.liferay.site.cms.site.initializer.internal.display.context.test.ViewRecycleBinSectionDisplayContextTest](https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1239894/testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewRecycleBinSectionDisplayContextTest) - [Failed 0 of last 24](https://testray.liferay.com/home/-/testray/case_results/300024044/history)
[testGetAdditionalProps](https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1239894//testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewRecycleBinSectionDisplayContextTest/testGetAdditionalProps)

thanks

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-29

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-29.liferay.com/job/test-portal-source-format/7201/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/7201/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 6 minutes</h3><p><span>Ran com.liferay.source.formatter at released version <a href="https://repository.liferay.com/nexus/content/repositories/liferay-public-releases/com/liferay/com.liferay.source.formatter/1.0.1531">1.0.1531</a>.</span><br/><em>*The <a href="https://github.com/liferay/liferay-portal/commits/master/modules/util/source-formatter">latest version</a> has not been released.</em></p><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/92c186705ac2d036c03d6a3175b188d7d2679fd9">92c186705ac2d036c03d6a3175b188d7d2679fd9</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/oliv-yu/liferay-portal/tree/LPD-40364-cms-fds-dropzone-combined">LPD-40364-cms-fds-dropzone-combined</a><br/>Branch GIT ID: <a href="https://github.com/oliv-yu/liferay-portal/commit/dade0af2793e998e9616c0f3dd32902b81e116ec">dade0af2793e998e9616c0f3dd32902b81e116ec</a></p>1 out of 1 jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-source-format/7201/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/7201/jenkins-report.html">here</a>.</h5></details></html>

oliv-yu commented:
Rebased! Also removed the FDS changes, as frontend infra will be making their own.

There was also an issue that the folder page on the content side would support drag and drop, so the latest commit is to address that.

oliv-yu commented:
ci:test:cms

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>cms</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/7414/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7414/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:cms - 2 out of 5 jobs passed in 1 hour</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/1617">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1357401_7414</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/92c186705ac2d036c03d6a3175b188d7d2679fd9">92c186705ac2d036c03d6a3175b188d7d2679fd9</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f7563c9bdefe1217124a74859cc5c3d7eaf1a93c">f7563c9bdefe1217124a74859cc5c3d7eaf1a93c</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/303272499">EE Development Acceptance (master) - 1616 - 2025-08-27[08:34:03]</a></p><details><summary><strong>ci:test:cms - 2 out of 5 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/7414/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>js-unit<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1247568/">js-unit/0/0</a></li></ul></li><li>modules-integration-postgresql163<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1196081/"><strike>modules-integration-postgresql163/0/0</strike></a></li></ul></li><li>playwright-js-tomcat101-postgresql163<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1420459/">playwright-js-tomcat101-postgresql163/0/0</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1247623/"><strike>playwright-js-tomcat101-postgresql163/1/0</strike></a></li></ul></li></ul></div><details><summary><strong>2 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7414/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1196081//consoleText">modules-integration-postgresql163/0/0</a><ol><li><details><summary><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1196081/testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewRecycleBinSectionDisplayContextTest">com.liferay.site.cms.site.initializer.internal.display.context.test.ViewRecycleBinSectionDisplayContextTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/303354740/history">Failed 0 of last 24</a></summary><div><ol><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1196081//testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewRecycleBinSectionDisplayContextTest/testGetAdditionalProps">testGetAdditionalProps</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1247623//consoleText">playwright-js-tomcat101-postgresql163/1/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-08/test-1-35/test-portal-acceptance-pullrequest(master)/7414/playwright-js-tomcat101-postgresql163/1/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1247623/testReport/site-cms-site-initializer_main_contents.spec/ts">site-cms-site-initializer/main/contents.spec.ts</a></summary><div><ol><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1247623//testReport/site-cms-site-initializer_main_contents.spec/ts/Create_a_picklist_from_the_structure_builder_by_opening_other_tab">Create a picklist from the structure builder by opening other tab</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1247623/testReport/site-cms-site-initializer_main_all.spec/ts">site-cms-site-initializer/main/all.spec.ts</a></summary><div><ol><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1247623//testReport/site-cms-site-initializer_main_all.spec/ts/Dragging_and_dropping_files_into_the_data_set_opens_upload_modal">Dragging and dropping files into the data set opens upload modal</a></li></ol></div></details></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

Mikellorza commented:
ci:report:303468351

liferay-continuous-integration commented:
Test suite default has been triggered on http://test-1-28

Mikellorza commented:
@oliv-yu include my commits to this PR, also  my old commits. This changes fix the conflict and the backend test: https://github.com/oliv-yu/liferay-portal/pull/110

oliv-yu commented:
Ran CMS tests w/ FDS changes here https://github.com/oliv-yu/liferay-portal/pull/111 and it looks like the drag and drop works and the `ViewRecycleBinSectionDisplayContextTest` passes 😌

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-37

oliv-yu commented:
ci:test:cms

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest(master)/13082">test-portal-acceptance-pullrequest(master)#13082</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/13082/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>cms<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6999">liferay-content-management#6999</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/306797467">[master] ci:test:cms - oliv-yu &gt; liferay-content-management - PR#6999 - 2025-09-02[11:30:01]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>306797467</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest(master)/13082/">test-portal-acceptance-pullrequest(master)#13082</a><br/></div></div>

oliv-yu commented:
ci:test:sf

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-31

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-31.liferay.com/job/test-portal-source-format/7641/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/7641/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><p><span>Ran com.liferay.source.formatter at released version <a href="https://repository.liferay.com/nexus/content/repositories/liferay-public-releases/com/liferay/com.liferay.source.formatter/1.0.1533">1.0.1533</a>.</span><br/><em>*The <a href="https://github.com/liferay/liferay-portal/commits/master/modules/util/source-formatter">latest version</a> has not been released.</em></p><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/da2aa2cba405295dd525d5bceaf6b365dfdad5cc">da2aa2cba405295dd525d5bceaf6b365dfdad5cc</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/oliv-yu/liferay-portal/tree/LPD-40364-cms-fds-dropzone-combined">LPD-40364-cms-fds-dropzone-combined</a><br/>Branch GIT ID: <a href="https://github.com/oliv-yu/liferay-portal/commit/6394aab6318bd9dfb282ffd8d59ce8632ee6da8b">6394aab6318bd9dfb282ffd8d59ce8632ee6da8b</a></p>1 out of 1 jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-31.liferay.com/job/test-portal-source-format/7641/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/7641/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-31.liferay.com/job/test-portal-source-format/7641">test-portal-source-format#7641</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/7641/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6999">liferay-content-management#6999</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/307028809">[master] ci:test:sf - oliv-yu &gt; liferay-content-management - PR#6999 - 2025-09-02[15:13:00]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>307028809</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-31.liferay.com/job/test-portal-source-format/7641/">test-portal-source-format#7641</a><br/></div></div>

liferay-continuous-integration commented:
Test suite cms has been triggered on http://test-1-37

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>cms</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/24053/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/24053/jenkins-report.html">here</a>.</b></p>

oliv-yu commented:
Hey @Mikellorza ! I rebased the changes on master, although I suspect there might be some integration test fails 🫠 ... if there are, can you help address them? Thank you!

liferay-continuous-integration commented:
<html><h3>:x: ci:test:cms - 2 out of 5 jobs passed in 1 hour 20 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/1626">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1377401_24053</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/da2aa2cba405295dd525d5bceaf6b365dfdad5cc">da2aa2cba405295dd525d5bceaf6b365dfdad5cc</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f7563c9bdefe1217124a74859cc5c3d7eaf1a93c">f7563c9bdefe1217124a74859cc5c3d7eaf1a93c</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/303272499">EE Development Acceptance (master) - 1616 - 2025-08-27[08:34:03]</a></p><details><summary><strong>ci:test:cms - 2 out of 5 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/24053/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>js-unit<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1218018/">js-unit/0/0</a></li></ul></li><li>modules-integration-postgresql163<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200840/"><strike>modules-integration-postgresql163/0/0</strike></a></li></ul></li><li>playwright-js-tomcat101-postgresql163<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200839/">playwright-js-tomcat101-postgresql163/0/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/894020/"><strike>playwright-js-tomcat101-postgresql163/1/0</strike></a></li></ul></li></ul></div><details><summary><strong>2 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/24053/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200840//consoleText">modules-integration-postgresql163/0/0</a><ol><li><details><summary><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200840/testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewAllSectionDisplayContextTest">com.liferay.site.cms.site.initializer.internal.display.context.test.ViewAllSectionDisplayContextTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/303354728/history">Failed 0 of last 24</a></summary><div><ol><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200840//testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewAllSectionDisplayContextTest/testGetAdditionalProps">testGetAdditionalProps</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200840/testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewContentsSectionDisplayContextTest">com.liferay.site.cms.site.initializer.internal.display.context.test.ViewContentsSectionDisplayContextTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/303354732/history">Failed 0 of last 24</a></summary><div><ol><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200840//testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewContentsSectionDisplayContextTest/testGetAdditionalProps">testGetAdditionalProps</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200840/testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewFilesSectionDisplayContextTest">com.liferay.site.cms.site.initializer.internal.display.context.test.ViewFilesSectionDisplayContextTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/303354736/history">Failed 0 of last 24</a></summary><div><ol><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200840//testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewFilesSectionDisplayContextTest/testGetAdditionalProps">testGetAdditionalProps</a></li></ol></div></details></li><li>...</li></ol></div></li><li><div><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/894020//consoleText">playwright-js-tomcat101-postgresql163/1/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-09/test-1-37/test-portal-acceptance-pullrequest(master)/24053/playwright-js-tomcat101-postgresql163/1/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/894020/testReport/site-cms-site-initializer_main_files.spec/ts">site-cms-site-initializer/main/files.spec.ts</a></summary><div><ol><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/894020//testReport/site-cms-site-initializer_main_files.spec/ts/There_is_a_View_action_for_items_the_Files_section">There is a View action for items the Files section</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/894020/testReport/site-cms-site-initializer_main_contents.spec/ts">site-cms-site-initializer/main/contents.spec.ts</a></summary><div><ol><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/894020//testReport/site-cms-site-initializer_main_contents.spec/ts/Can_translate_a_content_with_a_Select_field">Can translate a content with a Select field</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/894020/testReport/site-cms-site-initializer_main_all.spec/ts">site-cms-site-initializer/main/all.spec.ts</a></summary><div><ol><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/894020//testReport/site-cms-site-initializer_main_all.spec/ts/Can_view_Share_modal_for_added_content">Can view Share modal for added content</a></li></ol></div></details></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/24053">test-portal-acceptance-pullrequest(master)#24053</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/24053/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>cms<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6999">liferay-content-management#6999</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/307166180">[master] ci:test:cms - oliv-yu &gt; liferay-content-management - PR#6999 - 2025-09-02[16:00:04]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>307166180</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/24053/">test-portal-acceptance-pullrequest(master)#24053</a><br/></div></div>

liferay-continuous-integration commented:
Test suite sf has been triggered on http://test-1-31

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-31.liferay.com/job/test-portal-source-format/7648/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/7648/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-31.liferay.com/job/test-portal-source-format/7648">test-portal-source-format#7648</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-31.liferay.com/userContent/jobs/test-portal-source-format/builds/7648/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-content-management/liferay-portal/pull/6999">liferay-content-management#6999</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/307192882">[master] ci:test:sf - oliv-yu &gt; liferay-content-management - PR#6999 - 2025-09-02[17:59:32]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>307192882</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-31.liferay.com/job/test-portal-source-format/7648/">test-portal-source-format#7648</a><br/></div></div>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>cms</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/13064/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/13064/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:cms - 2 out of 5 jobs passed in 49 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/1626">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1297401_13064</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/da2aa2cba405295dd525d5bceaf6b365dfdad5cc">da2aa2cba405295dd525d5bceaf6b365dfdad5cc</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f7563c9bdefe1217124a74859cc5c3d7eaf1a93c">f7563c9bdefe1217124a74859cc5c3d7eaf1a93c</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/303272499">EE Development Acceptance (master) - 1616 - 2025-08-27[08:34:03]</a></p><details><summary><strong>ci:test:cms - 2 out of 5 jobs PASSED</strong></summary><div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest(master)/13064/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>js-unit<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200898/">js-unit/0/0</a></li></ul></li><li>modules-integration-postgresql163<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200901/"><strike>modules-integration-postgresql163/0/0</strike></a></li></ul></li><li>playwright-js-tomcat101-postgresql163<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1218081/">playwright-js-tomcat101-postgresql163/0/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1496639/"><strike>playwright-js-tomcat101-postgresql163/1/0</strike></a></li></ul></li></ul></div><details><summary><strong>2 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/13064/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200901//consoleText">modules-integration-postgresql163/0/0</a><ol><li><details><summary><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200901/testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewAllSectionDisplayContextTest">com.liferay.site.cms.site.initializer.internal.display.context.test.ViewAllSectionDisplayContextTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/303354728/history">Failed 0 of last 24</a></summary><div><ol><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200901//testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewAllSectionDisplayContextTest/testGetAdditionalProps">testGetAdditionalProps</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200901/testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewContentsSectionDisplayContextTest">com.liferay.site.cms.site.initializer.internal.display.context.test.ViewContentsSectionDisplayContextTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/303354732/history">Failed 0 of last 24</a></summary><div><ol><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200901//testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewContentsSectionDisplayContextTest/testGetAdditionalProps">testGetAdditionalProps</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200901/testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewFilesSectionDisplayContextTest">com.liferay.site.cms.site.initializer.internal.display.context.test.ViewFilesSectionDisplayContextTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/303354736/history">Failed 0 of last 24</a></summary><div><ol><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1200901//testReport/com.liferay.site.cms.site.initializer.internal.display.context.test/ViewFilesSectionDisplayContextTest/testGetAdditionalProps">testGetAdditionalProps</a></li></ol></div></details></li><li>...</li></ol></div></li><li><div><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1496639//consoleText">playwright-js-tomcat101-postgresql163/1/0</a> - <a href="https://storage.cloud.google.com/testray-results/2025-09/test-1-29/test-portal-acceptance-pullrequest(master)/13064/playwright-js-tomcat101-postgresql163/1/0/playwright-report/index.html">Playwright Report</a><ol><li><details><summary><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1496639/testReport/site-cms-site-initializer_main_contents.spec/ts">site-cms-site-initializer/main/contents.spec.ts</a></summary><div><ol><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1496639//testReport/site-cms-site-initializer_main_contents.spec/ts/Can_translate_a_content_with_a_Select_field">Can translate a content with a Select field</a></li></ol></div></details></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>
